msdsn/conn_str.go

```diff
@@ -72,6 +74,15 @@ type Config struct {
 	ConnTimeout time.Duration // Use context for timeouts.
 	KeepAlive   time.Duration // Leave at default.
 	PacketSize  uint16
+
+	// Kerberos authentication fields
+
+	Krb5ConfFile      string
```

Comment:
The config should hold the actual file, not a pointer to a file, so the driver can use it without accessing the OS fs at all.

Reply:
This variable holds path to the file and have renamed the variable.
The GOKRB package needs a path to file and not actual file